packages/volto-slate/src/editor/plugins/StyleMenu/style.less

```diff
@@ -0,0 +1,48 @@
+/* Demo styles */
+
+.green-block-text {
+  color: green;
+}
+
+.underline-block-text {
+  text-decoration: underline;
+}
+
+.cool-inline-text {
+  font-style: italic;
+  font-weight: bold;
+}
+
+.red-inline-text {
+  color: red;
+}
```

Comment:
Hm.. yes, they are not needed since we also removed the demo configuration in [d39cb17](https://github.com/plone/volto/pull/3616/commits/d39cb17fd0ce2b203d13a73208aa806a235a4092)